reformat_site_data.R

```diff
@@ -0,0 +1,12 @@
+reformat_site_data <- function(data){
+  for (x in length(data)) {
```

Comment:
`x in length(data)` potentially could be problematic mainly because it would run the loop once and for the x equal to the length of that list. An alternative could be something like this :
`x in seq_len(length(data))`
which `seq_len(length(data))` goes from 1 to the length of the list